JPS_Ontology/ontology/ontoclassschedule/OntoClassSchedule.csv

```diff
@@ -0,0 +1,19 @@
+Source,Type,Target,Relation,Domain,Range,Quantifier,Comment,Defined By,Label
+OntoClassSchedule,TBox,https://www.theworldavatar.com/kg/ontoclassschedule,https://www.w3.org/2007/05/powder-s#hasIRI,,,,,,
+OntoClassSchedule,TBox,1,http://www.w3.org/2002/07/owl#versionInfo,,,,,,
+OntoClassSchedule,TBox,"OntoClassSchedule is designed to represent and manage class schedules in an educational setting. It captures the relationships between sessions, courses, and various types of course offerings.",http://www.w3.org/2000/01/rdf-schema#comment,,,,,,
+OntoClassSchedule,TBox,http://www.w3.org/2006/time#,http://www.w3.org/2002/07/owl#imports,,,,,,
+Session,Class,,,,,,"A specific class meeting or event, capturing details like timing and association with a particular course.",http://www.theworldavatar.com/ontology/ontoclassschedule/OntoClassSchedule.owl,
+Course,Class,,,,,,"A structured unit of study or training, encompassing various sessions and topics.",http://www.theworldavatar.com/ontology/ontoclassschedule/OntoClassSchedule.owl,
+Laboratory,Class,Session,IS-A,,,,"A hands-on, practical session focused on experiments or activities, typically associated with scientific or technical subjects.",http://www.theworldavatar.com/ontology/ontoclassschedule/OntoClassSchedule.owl,
+Tutorial,Class,Session,IS-A,,,,"An interactive session designed for detailed discussion, problem-solving, or study on specific topics.",http://www.theworldavatar.com/ontology/ontoclassschedule/OntoClassSchedule.owl,
+Seminar,Class,Session,IS-A,,,,"A presentation or discussion-based session where specific topics are explored in-depth, often with guest speakers or experts.",http://www.theworldavatar.com/ontology/ontoclassschedule/OntoClassSchedule.owl,
+Lecture,Class,Session,IS-A,,,,"A formal instructional session where content is delivered to students, primarily through speech and visual aids.",http://www.theworldavatar.com/ontology/ontoclassschedule/OntoClassSchedule.owl,
+hasSession,Object Property,,,,Session,exactly 1,,http://www.theworldavatar.com/ontology/ontoclassschedule/OntoClassSchedule.owl,
+hasDayOfWeek,Object Property,,,Session,https://www.theworldavatar.com/kg/ontobim/Room,exactly 1,"Indicates the specific day of the week on which a session occurs, such as Monday or Tuesday.",http://www.theworldavatar.com/ontology/ontoclassschedule/OntoClassSchedule.owl,
```

Comment:
Surely the range is wrong here? It should be a "day of the week", not a "room"?

Reply:
Fixed the range of hasDayOfWeek to http://www.w3.org/2006/time#DayOfWeek.